pkg/skaffold/yamltags/tags_test.go

```diff
@@ -64,7 +64,7 @@ func TestProcessStructRequired(t *testing.T) {
 			wantErr: false,
 		},
 		{
-			name: "missng some",
+			name: "missing some",
```

Comment:
Can we add a test for `processTags` or `ValidateStructs` with an invalid tag i.e. not "oneOf", "required"

Reply:
Good idea! Done.